subjects/isthesquareachild/README.md

```diff
@@ -0,0 +1,48 @@
+## is-the-square-a-child
```

Comment:
i dont understand the naming of the exercise ... Why child?

Reply:
I will change it, just I did the same title that @mohamedLazyBob set in issue part